rcsbsearchapi/schema.py

```diff
@@ -12,15 +12,15 @@
 from .search import Attr
 from .const import STRUCTURE_ATTRIBUTE_SCHEMA_URL, CHEMICAL_ATTRIBUTE_SCHEMA_URL, SEARCH_SCHEMA_URL, STRUCTURE_ATTRIBUTE_SEARCH_SERVICE, CHEMICAL_ATTRIBUTE_SEARCH_SERVICE
 
-
```

Comment:
I think this extra line is needed for pylinting...

Reply:
I must have accidentally deleted that, I'll add it back in!